src/screens/Anchor.js

```diff
@@ -224,6 +224,14 @@ const AnchorPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="anchor.gap">
+          <Description>The gap between the label and icon.</Description>
+          <PropertyValue type="string">
+            <Example defaultValue>"small"</Example>
+            <Example>"xsmall"</Example>
+          </PropertyValue>
```

Comment:
I added an additional description and more examples so the user knows what they can pass in without having to look at the grommet theme code
```suggestion
          <PropertyValue type="string">
            <Description>
                T-shirt sizing based off the theme or a specific size in px, em,
                etc.
            </Description>
            <Example>"none"</Example>
            <Example>"xsmall"</Example>
            <Example defaultValue>"small"</Example>
            <Example>"medium"</Example>
            <Example>"large"</Example>
            <Example>"xlarge"</Example>
            <Example>"20 px"</Example>
          </PropertyValue>
```